### PR TITLE
Committing changes

### PR DESCRIPTION
**Pull request by VivekSharma-MSFT**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | yes                               |
| New feature?    | no                              |
| New sample?     | no                               |
| Related issues? | fixes #X, partially #Y, mentioned in #Z |

## What's in this Pull Request?

Updated the code to remove the dependency over hardcoded teamId and channelId.
Updated the read me and appsetting.json files.

### CODE REVIEW

File: samples/msgext-bot-SPUploader/SPUploader/appsettings.json [hunk 1]
```diff
@@ -9,13 +9,14 @@
   "SPTenantId": "<<SHAREPOINT-TENANT-ID>>",
   "SharePointClientID": "<<SHAREPOINT-CLIENT-ID>>",
   "SharePointClientSecret": "<<SHAREPOINT-CLIENT-SECRET>>",
-  "SharePointEndPoint": "<<YOUR-SHAREPOINT-DOMAIN.SHAREPOINT.COM>>",
-  "SharePointResource": "<<YOUR-SHAREPOINT-RESOURCE>>",
-  "SharePointBaseURL": "<<YOUR-SHAREPOINT-DOMAIN.SHAREPOINT.COM/YOUR-APP-NAME>>",
+  "SharePointEndPoint": "<<YOUR-SHAREPOINT-DOMAIN.SHAREPOINT.COM eg: xxxxxxxxx.SHAREPOINT.COM>>",
+  "SharePointResource": "00000003-0000-0ff1-ce00-000000000000",
```
Comment (dipja) [inline samples/msgext-bot-SPUploader/SPUploader/appsettings.json:13]: is it a constant for any tenant or any organisation?

File: samples/msgext-bot-SPUploader/README.md [hunk 2]
```diff
@@ -83,38 +83,48 @@ Step 1: Setup bot in Service
...
 	* Navigate to "API permissions" blade on the left-hand side.
 	* Add following permissions to the application.
-		* Application permissions - User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All,
-									GroupMember.Read.All, Group.Read.All, GroupMember.ReadWrite.All, Group.ReadWrite.All,Directory.AccessAsUser.All
-		* Delegated permissions - User.Read (enabled by default)
+	* Application permissions - User.Read.All and GroupMember.Read.All
+	* Delegated permissions - User.Read (enabled by default)
 
-<img src="./Docs/Images/APIPermissions.PNG" alt="API Permissions" width="1000" height="500">
+<img src="./Docs/Images/ApiPermissions.PNG" alt="API Permissions" width="1000" height="500">
```
Comment (dipja) [inline samples/msgext-bot-SPUploader/README.md:89]: This image is not loading, can you fix the image src / name

File: samples/msgext-bot-SPUploader/README.md [hunk 3]
```diff
@@ -83,38 +83,48 @@ Step 1: Setup bot in Service
...
-	* Go to  https//{SharePointDomain}/_layouts/15/appinv.aspx and serach with ClientID we generated earlier. The application will fetch all other details based on your ClientID.
-
-<img src="./Docs/Images/GrantPermission.PNG" alt="Bot Channels" width="1000" height="500">
-
 4. Create document library
-	* Create Two document libraries to upload the files into sharepoint doc library
+	* Create Two document libraries to upload the files into sharepoint doc library.
+	* You need to mention these document libraries in appsetting.json as StagingFolder and ApprovedFolder.
 
+<img src="./Docs/Images/CreateDocumentLibrary.png" alt="Create Document Library" width="1000" height="500">
```
Comment (dipja) [inline samples/msgext-bot-SPUploader/README.md:127]: This image is not loading in preview, can you fix the image src / name?

File: samples/msgext-bot-SPUploader/SPUploader/appsettings.json (hunk 1 above)
Comment (VivekSharma-MSFT) [reply samples/msgext-bot-SPUploader/SPUploader/appsettings.json:13]: yes, it is constant for all.
https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins

**Review by dipja — APPROVED** [review]

File: samples/msgext-bot-SPUploader/README.md (hunk 2 above)
Comment (VivekSharma-MSFT) [reply samples/msgext-bot-SPUploader/README.md]: Resolved the issue

File: samples/msgext-bot-SPUploader/README.md (hunk 3 above)
Comment (VivekSharma-MSFT) [reply samples/msgext-bot-SPUploader/README.md]: Resolved

**Pull request merged**